ctk/common/src/main/java/org/codice/compliance/utils/sign/SimpleSign.java

```diff
@@ -56,16 +56,53 @@
 import org.opensaml.xmlsec.signature.support.SignatureValidator;
 import org.opensaml.xmlsec.signature.support.provider.ApacheSantuarioSignatureValidationProviderImpl;
 
+// NOTE: "DDF Difference" means something that is different in the DDF version of this class.
+// DDF Difference: The {@code resignAssertion} and {@code forceSignSamlObject} methods were removed
+// since it was not needed.
 public class SimpleSign {
 
+  static {
+    OpenSAMLUtil.initSamlEngine();
+  }
+
+  private final String rsaAlgoUri;
+  private final String rsaAlgoJce;
+  private final String dsaAlgoUri;
+  private final String dsaAlgoJce;
+
   private final SystemCrypto crypto;
 
+  // DDF Difference: Uses SHA1 for the hashing mechanism when signing by default instead of SHA256.
   public SimpleSign() throws IOException {
     crypto = new SystemCrypto(getCurrentSPHostname());
+    rsaAlgoUri = WSS4JConstants.RSA;
+    rsaAlgoJce = JCEMapper.translateURItoJCEID(rsaAlgoUri);
+    dsaAlgoUri = WSS4JConstants.DSA;
+    dsaAlgoJce = JCEMapper.translateURItoJCEID(dsaAlgoUri);
+  }
+
+  // DDF Difference: The signing algorithm is not configurable in DDF.
+  public SimpleSign(String rsaAlgoUri, String dsaAlgoUri) throws IOException {
```

Comment:
❓ Should we just pass in one algorithm instead of two? Cause we'll just be using the one